Sources/Client/Client/Client+Setup.swift

```diff
@@ -162,10 +162,10 @@ extension Client {
         }
         
         if logOptions.isEnabled {
-            ClientLogger.logger(user.isAnonymous ? "👺" : "👤", "", user.isAnonymous ? "Anonymous" : "\(user.name): \(user.id)")
+            ClientLogger.logger(user.isAnonymous ? "👺" : "👤", "", .info, user.isAnonymous ? "Anonymous" : "\(user.name): \(user.id)")
             
             if !user.isAnonymous {
-                ClientLogger.logger("🀄️", "", "Token: \(token)")
+                ClientLogger.logger("🀄️", "", .info, "Token: \(token)")
```

Comment:
`.info` is by default you can avoid this in many places.

Reply:
`.info` is by default so that we can be backwards-compatible. I wouldn't want to put a default argument tbh
Better to have all of them verbose